src/utils/renderHtml/escapeHtml.ts

```diff
@@ -0,0 +1,9 @@
+const escapeHtml = (html: string) =>
```

Comment:
sorry one more thing; I noticed the highlight doesn't appear when searching for things with escaped characters, though the search picks it up correctly

<img width="1010" alt="Screen Shot 2022-10-19 at 10 11 53 AM" src="https://user-images.githubusercontent.com/47064971/196716238-26c36ce7-b459-4a38-9659-10902e0e870b.png">

Reply:
Yup i ended up moving the escape to after we find the match and wrap it in `<mark>`